README.md

```diff
@@ -14,8 +14,8 @@ Instead of this.hourlySales[hour], I had these two incorrect attempts: ..
 randomCustomer() is within generateHourly(), which is inside render: function()
 
 #Style-Guide
-+Header, body, and footer style guide
-+background-color, font-color, font, and links included
++ Header, body, and footer style guide
```

Comment:
Nice README. Glad to see that you're improving it on every iteration of the assignment!